README.md

```diff
@@ -1 +1,15 @@
-# hw01
\ No newline at end of file
+# hw01
+
+## Biography
+
+![Profile Image](/profile.jpg)
```

Comment:
Nice insertion of this image here.  I don't think you need the "/" but it doesn't matter unless it stops working!